<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPServer</class>
 <widget class="QMainWindow" name="TCPServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TCPServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>175</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9999</width>
              <height>9999</height>
             </size>
            </property>
            <property name="text">
             <string>服务器监听地址：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboIP">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9999</width>
              <height>9999</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="duplicatesEnabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>127.0.0.1</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>171</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9999</width>
              <height>9999</height>
             </size>
            </property>
            <property name="text">
             <string>服务器监听端口：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPort">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9999</width>
              <height>9999</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>1200</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="editMsg">
          <property name="minimumSize">
           <size>
            <width>305</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnSend">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>9999</width>
            <height>9999</height>
           </size>
          </property>
          <property name="text">
           <string>服务器发送消息</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="actHostInfo"/>
   <addaction name="actClear"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actStart">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/620.bmp</normaloff>:/images/620.bmp</iconset>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
   <property name="toolTip">
    <string>开始监听</string>
   </property>
  </action>
  <action name="actStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/624.bmp</normaloff>:/images/624.bmp</iconset>
   </property>
   <property name="text">
    <string>停止监听</string>
   </property>
   <property name="toolTip">
    <string>停止监听</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本程序</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/212.bmp</normaloff>:/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
   <property name="toolTip">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actHostInfo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/110.JPG</normaloff>:/images/110.JPG</iconset>
   </property>
   <property name="text">
    <string>本机地址</string>
   </property>
   <property name="toolTip">
    <string>获取本机地址</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>TCPServer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
